dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/publish/files/otel-collector-config-jaeger.yaml

```diff
@@ -0,0 +1,25 @@
+receivers:
+ otlp:
+   protocols:
+     grpc:
+     http:
+
+exporters:
+ logging:
+ jaeger:
+   endpoint: "jaeger-all-in-one:14250"    
+   tls:
+      insecure: true
+
+processors:
+ batch:
```

Comment:
Did you consider setting the batch timeout to zero to avoid unnecessary waiting in the tests? See https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md for an example.